extensions/common/api/api-core/src/main/java/org/eclipse/dataspaceconnector/api/model/StringResponseDto.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright (c) 2022 T-Systems International GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       T-Systems International GmbH
+ *
+ */
+
+package org.eclipse.dataspaceconnector.api.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+@JsonDeserialize(builder = String.class)
+public class StringResponseDto extends BaseResponseDto {
```

Comment:
Maybe `IdResponseDto` could fit better?

Reply:
@paullatzelsperger yes, but the field in this class is called "id", so if serialized it could be only an id.
a generic `StringResponseDto` should give the possibility to set the field name. `IdResponseDto` seems to fit better

Reply:
Its ok for me 👍